Stdlib/Prelude.juvix

```diff
@@ -8,5 +8,6 @@ open import Stdlib.Data.Product public;
 open import Stdlib.Data.String public;
 open import Stdlib.Function public;
 open import Stdlib.System.IO public;
+open import Stdlib.Debug public;
```

Comment:
Do we want to import Stdlib.Debug in the prelude?

If we don't, then it's easier to ensure that no debugging functions are left in a program by just removing the import of Stdlib.Debug.

Reply:
I like the last argument; let's do it that way.

Reply:
Done.